src/arche/arche.py

```diff
@@ -106,15 +106,15 @@ def schema(self, schema_source):
     def get_items(
         source: Union[str, pd.DataFrame, RawItems],
         count: Optional[int],
-        start: Union[str, int],
+        start: Optional[str],
```

Comment:
```suggestion
        start: Optional[Union[str, int]],
```

Reply:
I changed to `Optional[str]` because of this error:
`src/arche/arche.py:119: error: Argument 3 to "CollectionItems" has incompatible type "Union[str, int, None]"; expected "Optional[str]"`